sed/loader/flash/loader.py

```diff
@@ -924,7 +925,7 @@ def read_dataframe(
 
         df, df_timed = self.parquet_handler(data_parquet_dir, **kwds)
 
-        metadata = self.parse_metadata() if collect_metadata else {}
+        metadata = self.parse_metadata(**kwds) if collect_metadata else {}
```

Comment:
Did you test this? You pass the **kwds dict to both parquet_handler and parse_metadata, neither of which accept **kwds arguments. I would expect this to always lead to an unexpected keyword argument exception

Reply:
I didn't test the loader function before and you were right. I have fixed this now